src/apps/morpho/utils/morpho-blue/Blue.maths.ts

```diff
@@ -0,0 +1,106 @@
+import { MarketParams, MarketState, UserPosition } from './interfaces';
+
+import { MulDiv, getConvertToAssets, getConvertToShares, mulDivDown, mulDivUp, pow } from 'evm-maths/lib/utils';
+import { wadDivUp, wadMulDown } from 'evm-maths/lib/wad';
+import { WAD } from 'evm-maths/lib/constants';
+
+export const OraclePriceAbi = [
+  {
+    constant: true,
+    inputs: [],
+    name: 'price',
+    outputs: [
+      {
+        name: '',
+        type: 'uint256',
+      },
+    ],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function',
+  },
+];
+
+export class MorphoBlueMath {
+  static MAX_UINT_256: bigint = BigInt('0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff');
+
+  static ORACLE_PRICE_OFFSET = BigInt(36);
+  static ORACLE_PRICE_SCALE = pow(BigInt(1), this.ORACLE_PRICE_OFFSET, BigInt(1));
+  static VIRTUAL_ASSETS: bigint = BigInt(1);
+  static VIRTUAL_SHARES: bigint = BigInt(1e6);
```

Comment:
```suggestion
  static VIRTUAL_ASSETS = 1n;
  static VIRTUAL_SHARES = 1000000n;
```

You may already know this: this syntax exists and i personally find it easier

Reply:
<img width="788" alt="image" src="https://github.com/morpho-labs/zapper-studio/assets/75627352/c24fc72d-d416-438b-9b9c-3365bcf22e18">

As I got this error, I assumed it was impossible

[edit] -> ignoring the comment as is comes from Zapper codebase apparently